### ISSUE TITLE
Invalid Manifest when Exporting NeoForge pack to Curseforge

### ISSUE THREAD

**Issue by waddon1** [packwiz/packwiz#295]
When exporting a Packwiz Neoforge pack to Curseforge the modloader and version are not included in the manifest file making the pack invalid.

**Comment by johalun** [comment]
Yep I just noticed this when importing a pack to Prism Launcher.
